test/e2e/util.go

```diff
@@ -401,23 +403,21 @@ func pokeEndpointClientIP(clientContainer, protocol, targetHost string, targetPo
 // leverages a container running the netexec command to send a request to a target running
 // netexec on the given target host / protocol / port
 // returns either the name of backend pod or "Timeout" if the curl request timed out
```

Comment:
You probably want to remove this comment about returning "Timeout" here and for other functions using `parseNetexecResponse`

Reply:
Done.